web/src/utils/dns.ts

```diff
@@ -1,21 +1,16 @@
 import { Record } from '../api/ipfix';
 
 /**
- * setRecordsDNSTimes check from provided records matching DNS IDs and set DnsRequestTimeMs / DnsResponseTimeMs
- * As the set of records may not contains all the pairs, some informations may be missing
+ * setRecordsDNSTimes check from provided records matching DNS IDs and set DnsLatencyMs
```

Comment:
I guess this whole function isn't necessary anymore

Reply:
yes removed and remove its reference